src/figuresClasses.ts

```diff
@@ -1,11 +1,79 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  public shape: Figure['shape'] = 'triangle';
+
+  constructor(
+    public color: Figure['color'],
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    const sides: number[] = [a, b, c];
+
+    if (sides.some((s) => s <= 0)) {
+      throw new Error('Invalid sides');
```

Comment:
[CODE STYLE] - When throwing an error, provide a more detailed description in the error message to make it clear what the issue is. Instead of 'Invalid sides', you could use 'Triangle sides must be greater than zero'.